AirCasting/Utils/SystemSettings.swift

```diff
@@ -0,0 +1,32 @@
+// Created by Lunar on 15/07/2021.
+//
+
+import Foundation
+import SwiftUI
+
+protocol SettingsRedirectionProtocol {
```

Comment:
`Protocol` suffix is redundant here. Otherwise this is exactly what I've been asking for. Now it will be super easy to unit test this beahavior inside viewmodels and/or change implementation to something else if needed (perhaps iOS16 will give us some more options to do so and it will come handy, who knows, always good to have things well separated)